build/ci/community/check_all_checks.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export passed_k8s_checks=true
+export passed_openshift_checks=true
+
+did_all_checks_pass(){
+  community_operators_branch=$1
+  all_checks_passed=false
+  if [ "$(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -iv pass | wc -l)" -eq 0 ]
+  then
+    export all_checks_passed=true
+  fi
+  echo "$all_checks_passed"
+}
+
+passed_k8s_checks=$(did_all_checks_pass $community_operators_kubernetes_branch)
+passed_openshift_checks=$(did_all_checks_pass $community_operators_openshift_branch)
```

Comment:
why can't I see [these](https://github.com/operator-framework/community-operators/pull/4090#issuecomment-863226644) failures/runs?

Reply:
from the logs it seems they are using [this](https://github.com/openshift/ci-tools).
could we try to run it somewhere as well? if not in cloud then at least in our clusters?

Reply:
I think it should be in another ticket

Reply:
I think these missing tests is where we actually fail, so if we cover them in another ticket, that other ticket should be prioritized to be before this one.
if you think it is better, please open said ticket and write its number here, so we can resolve this

Reply:
done
CSI-3172